misc/config_tools/scenario_config/ivshmem_cfg_h.py

```diff
@@ -95,7 +95,7 @@ def generate_file(scenario_items, config):
 
     if vm_info.shmem.shmem_enabled == 'y':
         print("#include <ivshmem.h>", file=config)
-        print("#include <x86/pgtable.h>", file=config)
+        print("#include <asm/pgtable.h>", file=config)
```

Comment:
Do you regenerate the files under  misc\config_tools\data\generic_board\generic_code\<scenario_name> to reflect your changes of config_tools?